### ISSUE TITLE
Get Security Group in `public` command output

### ISSUE THREAD

**Issue by j4v** [duo-labs/cloudmapper#130]
The `public` command returns the following details for each EC2/ELB instance:
- account
- arn
- hostname
- ports
- type

It would be great if the SG affecting the instance was also included, in order to aid in remediation.

**Comment by 0xdabbad00** [comment]
This is resolved in #140

The output will now look like this:
```
{
    "account": "demo",
    "arn": "arn:aws:ec2:us-east-1:000000000000:instance/webalb/subnet-00000002",
    "hostname": "webalb",
    "ports": "443",
    "public_sgs": [
        "sg-00000003"
    ],
    "type": "elb"
}
```
where `public_sgs` is an array of all of the security groups making the ports public.  Unfortunately, it does not associate which Security Group is allowing which port range.

**Comment by j4v** [comment]
thanks Scott, this will be quite useful!
